examples/kubernetes/1.12/cilium-external-etcd.yaml

```diff
@@ -305,6 +326,10 @@ spec:
           valueFrom:
             configMapKeyRef:
               key: masquerade
+        - name: CILIUM_TOFQDNS_ENABLE_POLLER
```

Comment:
Did we accidentally snip 2 the name & optional lines for CILIUM_MASQUERADE here?

Reply:
Good catch. I will fix it now.

Reply:
fixed